osbs/core.py

```diff
@@ -357,22 +357,31 @@ def get_build_config_by_labels_filtered(self, label_selectors, filter_key, filte
         Returns a build config matching the given label selectors, filtering against
         another predetermined value. This method will raise OsbsException
         if not exactly one build config is found.
+        if a single build config is found matching the label selectors and no filter value is
+        provided, this method will return that build config.
+        if a single build config is found matching the label selectors but it does doesn't
+        match the filter value, this method will raise OsbsException instead of returning
+        that build config.
```

Comment:
Could these 5 lines be replaced by adding "after filtering" to the previous sentence, so it ends "if not exactly one build config is found after filtering"?

Reply:
Fixed.